src/core/public/doc_links/doc_links_service.ts

```diff
@@ -24,6 +24,7 @@ export class DocLinksService {
     const KIBANA_DOCS = `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/`;
     const FLEET_DOCS = `${ELASTIC_WEBSITE_URL}guide/en/fleet/${DOC_LINK_VERSION}/`;
     const PLUGIN_DOCS = `${ELASTIC_WEBSITE_URL}guide/en/elasticsearch/plugins/${DOC_LINK_VERSION}/`;
+    const APM_GET_STARTED_DOCS = `${ELASTIC_WEBSITE_URL}guide/en/apm/get-started/${DOC_LINK_VERSION}/`;
```

Comment:
> const APM_GET_STARTED_DOCS = `${ELASTIC_WEBSITE_URL}guide/en/apm/get-started/${DOC_LINK_VERSION}/`;

Note that for this new constant to be checked properly, you'll also need to add it to the build_docs.pl as I did for the KIBANA_DOCS one here: https://github.com/elastic/docs/pull/2132

If we end up needing links to the APM Server or APM agents books too, we might reconsider only having a single APM constant that is just `${ELASTIC_WEBSITE_URL}guide/en/apm/`

Reply:
@lcawl Thanks for the info, I opened https://github.com/elastic/docs/pull/2209 to add the new constant.